src/ext/background.js

```diff
@@ -16,59 +16,66 @@
 /* exported resetSpendVars */
 /* exported committedNavigation */
 /* exported cookiesChanged */
-/* exported CHL_CAPTCHA_DOMAIN */
-/* exported CHL_CLEARANCE_COOKIE */
-/* exported REDEEM_METHOD */
-/* exported RELOAD_ON_SIGN */
+/* exported chlCaptchaDomain */
+/* exported chlClearanceCookie */
+/* exported redeemMethod */
+/* exported reloadOnSign */
 /* exported spentTab, timeSinceLastResp, futureReload, sentTokens */
-/* exported DEV */
-/* exported COMMITMENTS_KEY */
-/* exported STORAGE_KEY_TOKENS, STORAGE_KEY_COUNT */
-/* exported SEND_H2C_PARAMS, MAX_TOKENS, SIGN_RESPONSE_FMT, TOKENS_PER_REQUEST */
+/* exported dev */
+/* exported commitmentsKey */
+/* exported storageKeyTokens, storageKeyCount */
+/* exported sendH2CParams, maxTokens, signResponseFMT, tokensPerRequest */
 /* exported CONFIG_ID */
-/* exported ISSUE_ACTION_URLS */
+/* exported issueActionUrls */
 /* exported LISTENER_URLS */
+/* exported getTarget */
+/* exported setFutureReload */
+
 "use strict";
 
 const LISTENER_URLS = "<all_urls>";
-/* Config variables that are reset in setConfig() depending on the header value that is received (see config.js) */
-let CONFIG_ID = ACTIVE_CONFIG["id"];
-let DEV = ACTIVE_CONFIG["dev"];
-let CHL_CLEARANCE_COOKIE = ACTIVE_CONFIG["cookies"]["clearance-cookie"];
-let CHL_CAPTCHA_DOMAIN = ACTIVE_CONFIG["captcha-domain"]; // cookies have dots prepended
-let CHL_VERIFICATION_ERROR = ACTIVE_CONFIG["error-codes"]["connection-error"];
-let CHL_CONNECTION_ERROR = ACTIVE_CONFIG["error-codes"]["verify-error"];
-let COMMITMENTS_KEY = ACTIVE_CONFIG["commitments"];
-let SPEND_MAX = ACTIVE_CONFIG["max-spends"];
-let MAX_TOKENS = ACTIVE_CONFIG["max-tokens"];
-let DO_SIGN = ACTIVE_CONFIG["sign"];
-let DO_REDEEM = ACTIVE_CONFIG["redeem"];
-let REDEEM_METHOD = ACTIVE_CONFIG["spend-action"]["redeem-method"];
-let HEADER_NAME = ACTIVE_CONFIG["spend-action"]["header-name"];
-let HEADER_HOST_NAME = ACTIVE_CONFIG["spend-action"]["header-host-name"];
-let HEADER_PATH_NAME = ACTIVE_CONFIG["spend-action"]["header-path-name"];
-let SPEND_ACTION_URLS = ACTIVE_CONFIG["spend-action"]["urls"];
-let SPEND_STATUS_CODE = ACTIVE_CONFIG["spending-restrictions"]["status-code"];
-let MAX_REDIRECT = ACTIVE_CONFIG["spending-restrictions"]["max-redirects"];
-let NEW_TABS = ACTIVE_CONFIG["spending-restrictions"]["new-tabs"];
-let BAD_NAV = ACTIVE_CONFIG["spending-restrictions"]["bad-navigation"];
-let BAD_TRANSITION = ACTIVE_CONFIG["spending-restrictions"]["bad-transition"];
-let VALID_REDIRECTS = ACTIVE_CONFIG["spending-restrictions"]["valid-redirects"];
-let VALID_TRANSITIONS = ACTIVE_CONFIG["spending-restrictions"]["valid-transitions"];
-let VAR_RESET = ACTIVE_CONFIG["var-reset"];
-let VAR_RESET_MS = ACTIVE_CONFIG["var-reset-ms"];
+let CONFIG_ID = 1;
+const getConfigId = () => CONFIG_ID;
```

Comment:
Should these go with the getters and setters below?

Reply:
Unfortunately not, they're being used line below, so need to be here, or further up, I'll add a newline here for readability